.github/workflows/pull-request.yml

```diff
@@ -25,11 +42,16 @@ jobs:
       - name: Setup Node.js
         uses: actions/setup-node@v3
         with:
-          node-version: 16
+          node-version: ${{ matrix.node-version }}
           cache: 'npm'
           registry-url: 'https://repo.backbase.com/artifactory/api/npm/npm-backbase/'
           scope: '@backbase'
 
+      - name: Use Angular ${{ matrix.angular-version }}
```

Comment:
I'm not sure that we can apply different Angular versions for the same code source. For example, if we upgrade the `main` branch to Angular v14 and start using standalone components, the app couldn't be built with a lower version.

My suggestion is to use a matrix parameter for repo branch names and specify a branch name in `actions/checkout` step. We should prepare a branch for each supported Angular version.

Reply:
That's a valid point. we don't need to run it for every pull request. If the event occurred on a particular repository branch, the workflow files should be in that branch's repository. I created the branches and split the workflows.